Content.Client/Lobby/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -612,6 +612,8 @@ public void RefreshAntags()
                 ("humanoid-profile-editor-antag-preference-no-button", 1)
             };
 
+            var clientBanManager = IoCManager.Resolve<JobRequirementsManager>();
+
```

Comment:
This is a field on the control already, you replaced the call to _requirements with this resolve instead AAAAA